src/states/node.rs

```diff
@@ -2411,8 +2411,13 @@ impl Node {
         }
 
         if self.tick_timer_token == token {
-            let tick_period = Duration::from_secs(TICK_TIMEOUT_SECS);
-            self.tick_timer_token = self.timer.schedule(tick_period);
+            if cfg!(feature = "use-mock-crust") {
+                trace!("{:?} not to carry out ticking during mock_crust test.",
+                       self);
+            } else {
+                let tick_period = Duration::from_secs(TICK_TIMEOUT_SECS);
+                self.tick_timer_token = self.timer.schedule(tick_period);
+            }
```

Comment:
But if we don't keep advancing time in the tests the ticking will stop, so there shouldn't be a loop?